test/specs/addToCartTest.smoke.js

```diff
@@ -1,14 +1,18 @@
-import Page from '../pageobjects/page.js'
 import LoginPage from '../pageobjects/login.page.js'
 import HomePage from '../pageobjects/home.page.js'
 import ProductPage from '../pageobjects/product.page.js'
 import CartPage from '../pageobjects/cart.page.js'
 
+
 describe('Add to cart', () => {
   it('should add product to cart from product page', async () => {
+    
+    console.log('Username:', global.username)
```

Comment:
No need to use `global` when referencing global vars

Reply:
removed